.travis.yml

```diff
@@ -18,6 +17,7 @@ env:
   - TASK=cpp_test
   # run tests/python
   - TASK=python_test
+  #- TASK=r_test
```

Comment:
Yes, according to @terrytangyuan The current config is not yet fully completed and there were some error in final setups, so it is not enabled